.github/workflows/build.yml

```diff
@@ -12,7 +12,9 @@ jobs:
         uses: actions/checkout@v3
       - name: Set imagemagick version
         id: imagemagick 
-        run: echo "version=$(cat ./.github/actions/imagemagick-version)" >> $GITHUB_OUTPUT
+        run: |
+          echo "version=$(cat ./.github/actions/imagemagick-version)" >> $GITHUB_OUTPUT
+          echo "release=$(cat ./.github/actions/release-version)" >> $GITHUB_OUTPUT
```

Comment:
I did not realize we had such generic variable names.
Please use something more precise like `imagemagick_version` & `imagemagick_release` that is also more consistent with the filenames